app/controllers/FieldTrip.java

```diff
@@ -26,18 +20,37 @@
 import models.fieldtrip.FieldTripNote;
 import play.data.binding.As;
 
-import javax.ws.rs.core.MediaType;
-import com.sun.jersey.api.client.*;
 import java.io.BufferedReader;
 import java.io.DataOutputStream;
 import java.io.InputStreamReader;
 import java.io.StringReader;
 import java.net.URL;
 import javax.net.ssl.HttpsURLConnection;
-import org.apache.commons.codec.digest.DigestUtils;
 
 
 public class FieldTrip extends Application {
+
+    /**
+     * Gson exclusion strategy designed to skip fields that could contribute to an overly large JSON response.
+     */
+    private static final ExclusionStrategy FIELD_TRIP_EXCLUSION_STRATEGY = new ExclusionStrategy() {
```

Comment:
This variable probably doesn't need to be in SCREAMING_SNAKE_CASE.

Reply:
Changed case in https://github.com/opentripplanner/otp-datastore/pull/7/commits/dbf308c28c0fe69ebb1a1316cea9812af8e6a77a.